openmetadata-ui/src/main/resources/ui/src/pages/UserListPage/UserListPageV1.tsx

```diff
@@ -182,7 +188,14 @@ const UserListPageV1 = () => {
   useEffect(() => {
     initialSetup();
     if (teamsAndUsers.includes(tab)) {
-      fetchUsersList(tab === GlobalSettingOptions.ADMINS || undefined);
+      if (location.search) {
+        const searchTerm = location.search.slice(1);
```

Comment:
Add comments here like what we are doing.

Reply:
Sure

Reply:
@aniketkatkar97 Make use or URLSearchParams: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams
Or you can use qs package installed